src/scss/constants.scss

```diff
@@ -102,7 +100,7 @@ $z-layers: (
   footer: $z-layer-sidebar,
   global-profile-icon: 1,
   user-avatar-status-dot: 1,
-  private-chat-input-search-icon: 1,
+  input-icon: 1,
```

Comment:
This should probably be `input-field-icon` or similar, and be included under a `// InputField` comment or similar.